packages/hardhat/contracts/WETHPool.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity >=0.6.0 <0.9.0;
+
+import "./Pool.sol";
+import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+
+contract WETHPool is Pool(ERC20(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)) {
```

Comment:
The last change I have in mind should solve this, pls poke me whenever you're free